### PR TITLE
Adjust calib-pressure-threshold for real gripper

### PR DESCRIPTION
**Pull request by wkentaro**

### CODE REVIEW

**Comment by wkentaro** [comment]
> Why do you want to calibrate threshold with grasping objects?

Just unexpectedly.

**Comment by pazeshun** [comment]
Ok, I understood

**Comment by wkentaro** [comment]
In last commit, I updated the code to get minimum threshold, please review it.

File: jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l [hunk 1]
```diff
@@ -1182,31 +1182,26 @@
...
+      (let ((min-pressure)
+            (topic (format nil "/gripper_front/limb/~a/pressure/state" (arm-to-str l/r)))
+            (hashkey (if (eq l/r :larm) :l-pressure-threshold :r-pressure-threshold)))
+        (dotimes (i 7)
+          (let ((pressure (send (one-shot-subscribe topic std_msgs::Float64) :data)))
+            (when (or (null min-pressure) (< pressure min-pressure)) (setq min-pressure pressure))
+            )
+          (unix::usleep (* 500 1000))
+          )
+        (sethash hashkey _hard-coded-variables (- min-pressure 8))
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l]: `(- min-pressure 10)`？

**Comment by pazeshun** [comment]
+1

**Pull request merged**